lab-jaren/lib/game-router.js

```diff
@@ -46,7 +46,24 @@ module.exports = function(router) {
     });
   });
 
-  // router.delete('/api/games', function(req, res) {
-  //
-  // });
+  router.delete('/api/games', function(req, res) {
+    let id = req.url.query.id;
+    if(!id) {
+      res.statusCode = 400;
+      res.end();
+      return;
+    }
+
+    storage.deleteItem('games', id)
+    .then(res => {
+      console.log('item deleted!');
+      res.statusCode = 204;
+      res.end();
+    })
+    .catch(err => {
+      res.statusCode = err.status;
+      res.end();
+    });
+    
+  });
 };
```

Comment:
When I do the DELETE method in httpie, it successfully deletes the item from storage but then the request times out and I get an UnhandledPromiseRejectionWarning and I couldn't figure out why.
![delete_httpie](https://cloud.githubusercontent.com/assets/19315840/21880651/f0fef4b0-d854-11e6-863c-df61acbe92b7.png)
![delete_warning](https://cloud.githubusercontent.com/assets/19315840/21880661/f9dab420-d854-11e6-834c-0acf6ca45c96.png)